dali/operators/generic/pad.cc

```diff
@@ -24,17 +24,16 @@
 namespace dali {
 
 DALI_SCHEMA(Pad)
-  .DocStr(R"code(Pads all samples with `fill_value` in the given `axes`,
-to match the biggest extent in the batch for those axes, or to match the minimum `shape`
-specified.
+  .DocStr(R"code(Pads all samples with the ``fill_value`` in the specified axes to match
+the biggest extent in the batch for those axes or to match the minimum shape specified.
 
-Supported types are integer and floating point numeric types.
+The supported types are integer and floating point numeric types.
```

Comment:
Doesn't it mean that all types are supported? If so, I'd remove this sentence, as it doesn't carry any information.

Reply:
Done